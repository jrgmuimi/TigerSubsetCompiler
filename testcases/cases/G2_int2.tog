let

in
    printint(440);
    printint(5555555);
    printint(440);
    printint(1111111);
    printint(440);
    printint(9999999)
end