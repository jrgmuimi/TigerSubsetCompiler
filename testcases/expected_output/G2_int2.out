MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

440555555544011111114409999999
